src/pretix/control/views/dashboards.py

```diff
@@ -169,6 +169,21 @@ def shop_state_widget(sender, **kwargs):
 
 
 @receiver(signal=event_dashboard_widgets)
+def checkin_widget(sender, **kwargs):
+    size = OrderPosition.objects.filter(order__event=sender, order__status='p', item__admission=True).count()
+    checked = Checkin.objects.filter(position__order__event=sender, position__order__status='p', position__item__admission=True).count()
```

Comment:
In theory, we could have more than one checkin per order position (although the API currently doesn't allow that), in which case this could return something like 20/10. Should we rather count OrderPositions in both cases and assert ``checkins__isnull=False`` in one case?